group/edwards25519/const.go

```diff
@@ -6,21 +6,31 @@ package edwards25519
 
 import (
 	"math/big"
-
-	"gopkg.in/dedis/kyber.v1/group/mod"
 )
 
 // prime modulus of underlying field = 2^255 - 19
 var prime, _ = new(big.Int).SetString("57896044618658097711785492504343953926634992332820282019728792003956564819949", 10)
 
 // prime order of base point = 2^252 + 27742317777372353535851937790883648493
-var primeOrder, _ = new(mod.Int).SetString("7237005577332262213973186563042994240857116359379907606001950938285454250989", "", 10)
+// XXX this should probably just be a big.Int
+var primeOrder, _ = new(big.Int).SetString("7237005577332262213973186563042994240857116359379907606001950938285454250989", 10)
```

Comment:
That comment is probably obsolete and I (or someone) just forgot to delete it.  primeOrder used to be defined as a nist.Int rather than a big.Int; looks like that's already fixed.